testdata/authorization/roles/User_1337/party_500000/roles.json

```diff
@@ -78,5 +78,9 @@
   {
     "Type": "altinn",
     "value": "HADM"
+  },
+  {
+    "Type": "altinn",
+    "value": "A0237"
```

Comment:
Det er mer naturlig et det er en av de andre brukerene som er ansvarlig revisor for Sophie sitt firma.

Reply:
Der så tenkte jeg å teste flyten hvor en bruker hadde tilgang til begge stegene, men kunne eventuelt brukt en annen revisorrolle?